src/client/src/services/model/trade.js

```diff
@@ -23,7 +24,7 @@ export default class Trade {
     tradeDate:Date,
     valueDate:string,
     status:TradeStatus,
-    isNew:boolean
+    isStateOfTheWorld:boolean
```

Comment:
@fdore  - could you please take a look? I don't think that the current implementation will support an initial payload, as all trades, even at the initial subscription, are processed individually?